lib/ledger_sync/adaptors/quickbooks_online/journal_entry/searcher.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+require_relative '../searcher'
+
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module JournalEntry
+        class Searcher < QuickBooksOnline::Searcher
+          def resources
+            @resources ||= begin
+              adaptor
+                .query(
+                  resource: 'journal_entry',
+                  limit: limit,
+                  offset: offset
+                )
+                .map do |c|
+                  LedgerSync::JournalEntry.new(
+                    ledger_id: c.fetch('Id')
```

Comment:
Can we pull in the other attributes while you're here?

Reply:
Yes, but this will get automated once we do serialisation.